etc/taskcluster/decision_task.py

```diff
@@ -648,6 +659,7 @@ def wpt_chunks(platform, make_chunk_task, build_task, total_chunks, processes,
                     --always-succeed \
                     webdriver \
                     | cat
+            """ + webgpu_script + """
                 ./mach filter-intermittents \
```

Comment:
We will want to include an equivalent filter-intermittents command in webgpu_script. That's what integrates our I-intermittent github label with the CI runs and allows us to ignore tests that have intermittent failures. We should also move the webgpu_script after the filter-intermittents command here.

Reply:
```
webgpu_script = """
                    | cat
                    time ./mach test-wpt _webgpu --release --processes $PROCESSES \
                    --headless --log-raw test-webgpu.log \
                    --log-errorsummary webgpu-errorsummary.log
                    ./mach filter-intermittents \
                    webgpu-errorsummary.log \
                    --log-intermittents webgpu-intermittents.log \
                    --log-filteredsummary filtered-webgpu-errorsummary.log \
                    --tracker-api default \
                    --reporter-api default
                """
```
@jdm does this look correct?

Reply:
Yep!